idl/github.com/uber/cadence/shared.thrift

```diff
@@ -735,6 +735,7 @@ struct PollForDecisionTaskRequest {
   10: optional string domain
   20: optional TaskList taskList
   30: optional string identity
+  40: optional bool isStickyTask
```

Comment:
I thought we decided that we need an enum, not a boolean.

Reply:
also, could it make sense to add the enum to the TaskList struct, instead of here?

Reply:
I agree it make sense to add that info to TaskList, changed and tested